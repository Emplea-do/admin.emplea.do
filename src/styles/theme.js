const Theme = {
  colors: {
    primary: 'green',
  },
}

export default Theme
