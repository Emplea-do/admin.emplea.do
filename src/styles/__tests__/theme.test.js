import theme from '../theme'

describe('theme', () => {
  test('application theme has no changes', () => {
    expect(theme).toMatchSnapshot()
  })
})
