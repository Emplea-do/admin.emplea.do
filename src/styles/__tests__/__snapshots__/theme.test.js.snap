// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`theme application theme has no changes 1`] = `
Object {
  "colors": Object {
    "primary": "green",
  },
}
`;
